containers/codespaces-linux/test-project/test.sh

```diff
@@ -3,6 +3,11 @@ cd $(dirname "$0")
 
 source test-utils.sh codespace
 
+echo '---'
+which python
+echo $PATH
+echo '---'
+
```

Comment:
Should we keep this?

Reply:
Remove? unless these logging statements are helpful